iroh-gateway/src/client.rs

```diff
@@ -231,13 +231,24 @@ impl<T: ContentLoader> Client<T> {
 }
 
 #[derive(Debug, Clone)]
-pub struct Request {
+pub struct IpfsRequest {
     pub format: ResponseFormat,
     pub cid: CidOrDomain,
     pub resolved_path: iroh_resolver::resolver::Path,
     pub query_file_name: String,
     pub download: bool,
     pub query_params: GetParams,
+    pub is_subdomain_mode: bool,
```

Comment:
Nitpick, can we make this `subdomain_mode`

Reply:
Done